### DISCUSSION TITLE
Which version of webpack is using latest version of set-value library?

### DISCUSSION THREAD

**Discussion by ramsha-tariq** [webpack/webpack#14368] [Q&A]
Hi,

The latest version of set-value is 4.1.0 (https://www.npmjs.com/package/set-value)

The project is using react-scripts v4.0.3, which is using webpack v4.44.2 which has set-value as the transitive dependency:

-- react-scripts@4.0.3
  -- webpack@4.44.2
    -- micromatch@3.1.10
      -- snapdragon@0.8.2
        -- base@0.11.2
          -- cache-base@1.0.1
            +-- **set-value@2.0.1**
            -- union-value@1.0.1
              -- **set-value@2.0.1**  deduped

The set-value v2.0.1 has the following vulnerability:
https://github.com/advisories/GHSA-4jqc-8m5r-9rpr

Does the latest version of webpack has updated version of set-value?

Thanks,

**Comment by alexander-akait** [comment]
Please ask this in `react-scripts` repo

**Reply by ramsha-tariq** [reply]
Once I can get the confirmation which webpack version is using the latest version of set-value library then will contact react scripts. Thanks!
